src/flb_input.c

```diff
@@ -137,7 +137,7 @@ int flb_input_event_type_is_metric(struct flb_input_instance *ins)
 
 int flb_input_event_type_is_trace(struct flb_input_instance *ins)
```

Comment:
this interface should be deprecated, now plugins do not belong to an event_type, but they can ingest different types.

Reply:
meaning: don't modify this file